tests/test_position_encoding.py

```diff
@@ -220,26 +216,14 @@ def test_encode_multiple_modalities():
         )
 
 
-@pytest.mark.parametrize("positioning", ["relative", "both"])
-def test_not_implemented_option(positioning):
+def test_fake_method_option():
     datetimes, geospatial_bounds, geospatial_coordinates = get_data()
-    with pytest.raises(NotImplementedError):
-        encode_modalities(
-            modalities_to_encode={"NWP": torch.randn(16, 1, 1, 1)},
+    with pytest.raises(AssertionError):
+        encode_position(
+            shape=[1, 1, 1, 1],
+            positioning="relative",
```

Comment:
I think `positioning` should be removed here as well, as `method="test_method"` is supposed to trigger the `AssertionError` - or am I missing something?

```suggestion
```

Reply:
Yes, you are right!